src/pages/host/list.jsx

```diff
@@ -1,16 +1,36 @@
-import React from "react";
+import { Link, useLoaderData } from "react-router-dom";
+import { getVans } from "../../api";
+import "../../server";
+import { loader as loadLogOut } from "../authentication";
+
+export async function loader({ request }) {
+  return { vans: await getVans(), prevPath: loadLogOut({ request }) };
+}
 
 function List() {
-  return (
+  const list = useLoaderData();
+  console.log(list.prevPath);
+  const loading = false;
+
+  return loading ? (
+    <div className="text-center py-52">Loading...</div>
+  ) : (
     <div className="bg-[#fff7ed] px-4 py-8 ">
-      <div className="flex justify-between">
-        <p className="font-bold">Your listed vans</p>
-        <button>View all</button>
-      </div>
-      <div className="py-12">
-        <p className="text-center text-red-800 font-bold">
-          You haven't rented any van yet!
-        </p>
+      <h2 className="font-bold text-4xl">Your listed vans</h2>
+      <div className="py-12 flex flex-col gap-4">
+        {list?.vans.map((item, idx) => (
```

Comment:
You can remove the idx variable as you not using it.

Reply:
I'm reusing it to hold the :id params